test/include.am

```diff
@@ -58,25 +58,25 @@ LOG_COMPILER = true
 TESTS_COMPILER = true
 
 test_runner_epoll: test/test.sh
-	test/test.sh -b EPOLL
+	$(srcdir)/test/test.sh -b EPOLL
```

Comment:
If you build elsewhere (mkdir $something; cd $something; ${oldpwd}/configure && make && make check), test.sh is still around with the sources, so it won't be found without this change.